src/CampaignAdmin.php

```diff
@@ -730,13 +730,9 @@ public function getCampaignCreateForm()
     }
 
     /**
-     * Save  handler
-     *
-     * @param array $data
-     * @param Form $form
-     * @return HTTPResponse
+     * Save handler
      */
-    public function save($data, $form)
+    public function save(array $data, Form $form): HTTPResponse
```

Comment:
`Security::PermissionFailure()` sometimes returns an array

Reply:
As noted in other PRs, it doesn't